vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -385,19 +395,35 @@ bool ViewViewManager::UpdateProperty(
     }
   }
 
-  if (auto view = pViewShadowNode->GetView().try_as<xaml::UIElement>()) {
+  return ret;
+}
+
+void ViewViewManager::OnPropertiesUpdated(ShadowNodeBase *node) {
+  auto *viewShadowNode = static_cast<ViewShadowNode *>(node);
+  auto panel = viewShadowNode->GetViewPanel();
+
+  if (panel.Background() == nullptr) {
+    // In XAML, a null background means no hit-test will happen.
+    // We actually want hit-testing to happen if the app has registered
+    // for mouse events, so detect that case and add a transparent brush.
```

Comment:
```suggestion
    // for mouse events, so detect that case and add a transparent background.
```

Reply:
@kmelmon that codeflow breaks github PRs especially those with suggestion tags

Reply:
dang it.  OK I'll stop using the codeflow UI to resolve stuff.  I wish our tools worked better.

Reply:
done